pkg/environment/api/organization_test.go

```diff
@@ -227,12 +227,13 @@ func TestCreateOrganizationMySQL(t *testing.T) {
 		expected    *proto.Organization
 		expectedErr error
 	}{
-		{
-			desc:        "err: ErrNoCommand",
-			setup:       nil,
-			req:         &proto.CreateOrganizationRequest{},
-			expectedErr: createError(statusNoCommand, localizer.MustLocalizeWithTemplate(locale.RequiredFieldTemplate, "command")),
-		},
+		// Deprecated
```

Comment:
I didn't add a unit test because as soon as we implement this on the new console, I'll reuse the current unit test, removing the deprecated `command` field.